### ISSUE TITLE
Installation fails on fresh install of Ubuntu 23.04

### ISSUE THREAD

**Issue by jpcapone** [itiligent/Guacamole-Install#23]
when launching the script I get the error:
`2-install-guacamole.sh FAILED. See /home/jpcapone/guac-setup/guacamole_setup.log`

When I review the log the last entry states:
`Held packages were changed and -y was used without --allow-change-held-packages.`
Not sure if thats whats causing the failure but please let me know if you would like any further information as I continue to troubleshoot.

**Comment by itiligent** [comment]
A bit more info about other packages and system origin etc if you can. On a fresh system I'm sure I've tested this when I added the tomcat9 package hold just for Ubuntu 23.x, coz I accidently added it to Debian as well.  
You could comment out the line and see if that makes any difference.. It should work ok without it, it was a line added as a preemtive fix when timcat 9 is depricated. I'll run another test to be sure.

**Comment by itiligent** [comment]
I did test this last month, but perhaps I missed it with keeping up with the nuances of several distros. I generally do most checking with LTS releases (which Ub23 is not), so these generally are better to use. 
 There was a line  `sudo apt-mark hold $TOMCAT_VERSION` in the 1-setup.sh that is there to stop tomcat9 getting upgraded at some point down the track. This was only a future preventative measure as tomcat 10 kills guacamole and I was worried that inadvertant upgrades will casue issues down the line. Anyway, without this line the setup runs fine on Ubuntu 23.x so I've removed it. All should work fine now.
